index.html

```diff
@@ -4,276 +4,209 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title> Web developers Portfolio  </title>
+  <title>Portfolio</title>
   <link rel="stylesheet" href="style.css">
-  <link rel="stylesheet" href="script.js">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
 </head>
 <body>
   
-  
     <header class="header">
-      <div class="menu-bar">
-        <a href="welcome" class="head">Welcome</a>
-        <nav class="menu-bar2">
-          <ul class="nav-menu" id="menu">
-            <a href="" onclick="" class="close-button">&times;</a>
-            <li ><a onclick="closeMenu()" href="#portfolio">Portfolio</a></li>
-            <li><a onclick="closeMenu()" href="#about">About</a></li>
-            <li><a onclick="closeMenu()" href="#contact">Contact</a></li>
-          </ul>
-          <div class="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-          </div>
-          </nav>
-      </div>
-        <nav class="menu-bar-desktop">
-          <ul class="nav-menu-desktop">
-            <!-- <a href="" onclick="" class="close-button">&times;</a> -->
-            <li ><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
-          </ul>
-        </nav>
-        
+      <!-- <a href="welcome" class="head">Welcome</a> -->
+      <nav class="menu-bar">
+        <img src="images/Normal Button/Enabled.png" alt=" menu-bar">
+      </nav>
     </header>
-    
     <section class="head-section">
     <div class="hey-there">
-<h1 class="hey-text">Hey there. I’m Mariane <br>
-  <span class="hey-text2"> I’m a software developer</span></h1>
+      <h2 class="hey-text">Hey there.<br> I’m Trevor.<br>
+      <span class="hey-text2"> I’m a software developer</span></h2>
```

Comment:
[OPTIONAL] Kindly make sure that this part looks exactly like the design in all screen sizes of the mobile version. There should be a line break between the `I'm a software` and `developer`, however, at screenwidths greater than 600px, they stay on the same line. Kindly update your app accordingly. :pray: 

### Your app below 600px 

![Screenshot from 2023-04-29 08-28-48](https://user-images.githubusercontent.com/65233860/235293300-afbc7380-3784-496c-94f1-db0c47c1c855.png)


### Your app above 600px

![Screenshot from 2023-04-29 08-27-20](https://user-images.githubusercontent.com/65233860/235293291-d2b80d90-79ba-49cf-90c3-db3df4f27b48.png)



### Actual design 


![Screenshot from 2023-04-29 08-25-55](https://user-images.githubusercontent.com/65233860/235293284-98106460-1240-4522-a2df-b68e972d72e5.png)